### PR TITLE
Add basic things to README.md

### PR DESCRIPTION
**Pull request by bashor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,77 @@
-# DefinitelyKotlin
\ No newline at end of file
+# External declarations for Kotlin/JS
+
+This repository contains sources of external declarations for Kotlin/JS.
+
+## How to use?
+
+### Bintray
+All libraries available in the maven repository https://bintray.com/kotlin/js-externals/
```
Comment (hhariri) [inline README.md:8]: All libraries are available

File: README.md [hunk 2]
```diff
@@ -1 +1,77 @@
...
+with group id `kotlin.js.externals` and artifact id `kotlin-js-<library-name>`.
+
+So, to use declarations for some library `foo` you need to add
```
Comment (hhariri) [inline README.md:11]: To use declarations from a library `foo`, you need to add

File: README.md [hunk 3]
```diff
@@ -1 +1,77 @@
...
+this repository to your repository list for your build tool and add a dependency to `kotlin-js-foo`.
```
Comment (hhariri) [inline README.md:12]: build too, and then add a dependency

File: README.md [hunk 4]
```diff
@@ -1 +1,77 @@
...
+        <groupId>kotlin.js.externals</groupId>
+        <artifactId>kotlin-js-foo</artifactId>
+        <version>$foo_version</version>
+    </dependency>
+</dependencies>
+```
+
+### NPM
+
+All libraries also available on npmjs.com under `@kotlin-externals` scope with the original name.
```
Comment (hhariri) [inline README.md:46]: All libraries are available on

File: README.md [hunk 5]
```diff
@@ -1 +1,77 @@
...
+I.e. to install declarations for the library `foo` you must use `@kotlin-externals/foo`:
```
Comment (hhariri) [inline README.md:47]: To install

File: README.md [hunk 6]
```diff
@@ -1 +1,77 @@
...
+
+```
+npm install --save-dev @kotlin-externals/foo
+```
+
+### How to choose version number?
+For a JS library version `X.Y.Z` use an external declarations library with latest `X.Y` version.
```
Comment (hhariri) [inline README.md:54]: For a JavaScript library of version `X.Y.Z`, use

File: README.md [hunk 7]
```diff
@@ -1 +1,77 @@
...
+
+
+> Sidenote:
+>
+> We have to use such version scheme to achieve next goals:
```
Comment (hhariri) [inline README.md:59]: This version scheme is required in order to:

File: README.md [hunk 8]
```diff
@@ -1 +1,77 @@
...
+> * Get simple mapping to/from original library version.
```
Comment (hhariri) [inline README.md:60]: Obtain simple

File: README.md [hunk 9]
```diff
@@ -1 +1,77 @@
...
+> * Get an ability to update external declarations
```
Comment (hhariri) [inline README.md:61]: Have the ability to

File: README.md [hunk 10]
```diff
@@ -1 +1,77 @@
...
+> (a version of external declarations library must have some part independent of a version of the original library).
+> * Don't use pre-release version by default (see https://docs.npmjs.com/misc/semver#prerelease-tags).
```
Comment (hhariri) [inline README.md:63]: Not use pre-release versions by default

File: README.md [hunk 11]
```diff
@@ -1 +1,77 @@
...
+> * Fit into three numbers (Semver/NPM's restriction).
```
Comment (hhariri) [inline README.md:64]: Follow the three number Semver/NPM restriction

File: README.md [hunk 12]
```diff
@@ -1 +1,77 @@
...
+
+## How to contribute?
+* [Report](https://github.com/Kotlin/js-externals/issues) any issue that you found in existing declarations.
```
Comment (hhariri) [inline README.md:67]: Report any issues you find

File: README.md [hunk 13]
```diff
@@ -1 +1,77 @@
...
+* Feel free to fix them by yourself and send PR.
```
Comment (hhariri) [inline README.md:68]: Or fix them and send a Pull Request!

File: README.md [hunk 14]
```diff
@@ -1 +1,77 @@
...
+* [Request](https://github.com/Kotlin/js-externals/issues) declarations that you missed.
```
Comment (hhariri) [inline README.md:69]: Request declarations that are missing

File: README.md [hunk 15]
```diff
@@ -1 +1,77 @@
...
+* Feel free to create them by yourself and send PR. (Don't hesitate to ask any question in [#javascript](https://kotlinlang.slack.com/messages/C0B8L3U69/) channel of [kotl.in/slack](https://kotl.in/slack))
```
Comment (hhariri) [inline README.md:70]: Or create them yourself and send a Pull Request. Please feel free to ask any questions in ...

File: README.md [hunk 16]
```diff
@@ -1 +1,77 @@
...
+
+## How to write declarations?
+
+To write declarations from scratch or to improve existing declarations
+you have to read about [external modifier](https://kotlinlang.org/docs/reference/js-interop.html#external-modifier) and [JS interop annotations](https://kotlinlang.org/docs/reference/js-modules.html#jsmodule-annotation).
+
+But the simplest way to get external declarations is converting TypeScript declarations (d.ts) to Kotlin [using ts2kt](https://kotlinlang.org/docs/tutorials/javascript/working-with-javascript.html#using-ts2kt-to-generate-header-files-for-kotlin).
```
Comment (hhariri) [inline README.md:77]: The simplest

**Review by hhariri — CHANGES_REQUESTED** [review]
Left some fixes inline

**Comment by bashor** [comment]
Thank you! 
Fixed and merged manually.

**Pull request closed without merging**